tests/Gperf-trace.c

```diff
@@ -235,6 +235,8 @@ main (int argc, char **argv)
 
   measure_init ();
 
+  doit ("default         ");
```

Comment:
no, the default changes depending on configuring with --enable-per-thread-cache, thus showing how the default performs make sense, doesn't it?